pkg/api/controller_test.go

```diff
@@ -2687,6 +2687,43 @@ func TestController_ObjectsGetObjectHandler(t *testing.T) {
 			t.Errorf("expected to get \"%s\" storage class, got %#v", expensiveString, properties)
 		}
 	})
+
+	t.Run("get object returns expected response with different etag", func(t *testing.T) {
+		newChecksum := `"11ee22ff33445566778899"`
+		eTagInput := "\"" + newChecksum + "\""
+		resp, err := clt.GetObjectWithResponse(ctx, repo, "main", &apigen.GetObjectParams{
+			Path:        "foo/bar",
+			IfNoneMatch: &eTagInput,
+		})
+		if err != nil {
+			t.Fatal(err)
+		}
+		if resp.HTTPResponse.StatusCode != http.StatusOK {
```

Comment:
You can use require.NotNil(resp.HTTP200) instead of this if clause

Reply:
Sure, but wouldn't it be even better to do `require.Equal(t, http.StatusOK, respList.StatusCode())`